kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/net/status/wifi/WifiAccessPoint.java

```diff
@@ -28,13 +28,23 @@
 @ProviderType
 public class WifiAccessPoint {
 
+    /** The Service Set IDentifier of the WiFi network */
```

Comment:
Javadocs on private fields will not be visible, please move them to the getter methods instead.